pages/content-runtime/src/messageHandler.ts

```
@@ -0,0 +1,166 @@
+
+  if (message.type === 'CHECK_READABILITY') {
+    console.log('[DEBUG] Checking readability');
+    // Implement readability check if needed
+    sendResponse({ type: 'READABILITY_RESULT', isReadable: true });
+    return true;
+  }
+
+  if (message.type === 'CAPTURE_SELECTION') {
+    console.log('[DEBUG] Capturing selection');
+    const selection = window.getSelection();
+    if (selection && selection.toString().trim()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)